Practice2/src/styles/components/top.css

```diff
@@ -0,0 +1,76 @@
+.page-top {
+  position: relative;
+  width: 100%;
+  height: 100%;
+}
+
+.top-img {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  z-index: -1;
+}
+
+.top-img-sub {
+  display: flex;
+  top: 312px;
+  margin: 53px auto;
+  padding-bottom: 132px;
+  justify-content: center;
+}
+
+.play-button {
+  background-color: Transparent;
+  background-repeat: no-repeat;
+  border: none;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin: -110px -36px;
+}
+
+.top-text {
+  text-transform: uppercase;
+  font-size: 36px;
+  line-height: var(--lh-heading-wide);
+  margin-right: 279px;
+  color: var(--c-warning);
+}
+
+.nav {
+  padding-top: 19px;
+}
+
+.nav-top {
+  display: flex;
+  padding: 48px var(--p-large) 0 var(--p-large);
+}
+
+.nav-icon {
+  padding-top: 19px;
+  margin-left: 148px;
+}
+
+.nav-icon-success {
+  margin-right: 43px;
+}
+
+.top-heading {
+  top: 163px;
+  font-size: var(--fs-heading-large);
+  line-height: var(--lh-heading-large);
+  text-align: center;
+  margin-top: 74px;
+  padding: 0 410px;
```

Comment:
Should not use padding with a very big value like that. Let try to find another way.